src/clock-weather-card.ts

```diff
@@ -227,6 +229,7 @@ export class ClockWeatherCard extends LitElement {
           <clock-weather-card-today-right-wrap-top>
             ${this.config.hide_clock ? weatherString : localizedTemp ? `${weatherString}, ${localizedTemp}` : weatherString}
             ${this.config.show_humidity && localizedHumidity ? html`<br>${localizedHumidity}` : ''}
+            ${this.config.show_apparent && apparentTemp ? html`<br>Feels like: ${localizedApparent}` : ''}
```

Comment:
`Feels like` should be localized

Reply:
should be localized now i think

Reply:
tested locally it works for english, i've put it in the EN file but no idea how to properly get it localized for the other languages.

Reply:
@pkissling how do you do it? or should i just run it through chatgpt?

Reply:
i would rather add the key to every translation json with the english translation, so that people can update it by raising a PR.

Reply:
Done